<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="uk.ac.sheffield.com1003.cafe.TestRecipeMultipleIngredientsDifferentMilkType" tests="1" skipped="0" failures="1" errors="0" timestamp="2024-03-18T21:30:56" hostname="Ruizhens-MacBook-Pro.local" time="0.001">
  <properties/>
  <testcase name="testRecipeMultipleIngredientsDifferentMilkType" classname="uk.ac.sheffield.com1003.cafe.TestRecipeMultipleIngredientsDifferentMilkType" time="0.001">
    <failure message="java.lang.AssertionError: Values should be different. Actual: uk.ac.sheffield.com1003.cafe.Recipe@2c242c21" type="java.lang.AssertionError">java.lang.AssertionError: Values should be different. Actual: uk.ac.sheffield.com1003.cafe.Recipe@2c242c21
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failEquals(Assert.java:187)
	at org.junit.Assert.assertNotEquals(Assert.java:163)
	at org.junit.Assert.assertNotEquals(Assert.java:177)
	at uk.ac.sheffield.com1003.cafe.TestRecipeMultipleIngredientsDifferentMilkType.testRecipeMultipleIngredientsDifferentMilkType(TestRecipeMultipleIngredientsDifferentMilkType.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:750)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
